app/src/main/java/org/keepgoeat/presentation/home/HomeActivity.kt

```diff
@@ -0,0 +1,65 @@
+package org.keepgoeat.presentation.home
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import org.keepgoeat.R
+import org.keepgoeat.databinding.ActivityHomeBinding
+import org.keepgoeat.presentation.home.adapter.HomeMyGoalAdapter
+import org.keepgoeat.util.binding.BindingActivity
+import org.keepgoeat.util.setVisibility
+
+class HomeActivity : BindingActivity<ActivityHomeBinding>(R.layout.activity_home) {
+
+    private val viewModel by viewModels<HomeViewModel>()
+    private lateinit var goalAdapter: HomeMyGoalAdapter
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        initLayout()
+    }
```

Comment:
```suggestion
    private val viewModel by viewModels<HomeViewModel>()
    private lateinit var goalAdapter: HomeMyGoalAdapter
    
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        initLayout()
    }
```

코드 통일성을 위해 줄바꿈을 이렇게 적용시켜보면 좋을 것 같아용! 
전역변수는 줄바꿈 없이 클래스명 바로 하단에 위치하고, 
처음에 작성되는 메서드 -> onCreate()는 전역변수에서 줄바꿈 하나 추가하고 바로 하단에 넣는 것입니당! 
그리고 전역변수가 없다면 클래스명 바로 하단에 줄바꿈 없이 메서드를 작성하시면 좋을 것 같네용!! 
@Dan2dani도 확인해주세용 🐰🐰🐰